src/Data/Repositories/SaltRepository.cs

```diff
@@ -0,0 +1,11 @@
+using NodeGuard.Data.Repositories.Interfaces;
+
+namespace NodeGuard.Data.Repositories;
+
+public class SaltRepository: ISaltRepository
```

Comment:
Salt cannot be a repository, it's not an entity in the system, it's just a `Constant` with a default hardcoded value.

Reply:
I know, I think I overengineered a bit because I make it this way with the idea of extending this in a future to a proper engine or component that actually handles better the salt :(